src/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTest.cs

```diff
@@ -2138,6 +2138,25 @@ public void GetAsync_ServerNeedsAuthAndNoCredential_StatusCodeUnauthorized()
             });
         }
 
+        [OuterLoop]
+        [Fact]
+        public async Task GetAsync_IdnHostName()
```

Comment:
Usually the naming for the test function should be like: A_B_C, A is the attribute what we are testing about, B is the condition to trigger the attribute, and C is the behavior outcome.

You can change to something like `GetAsync_IdnHostName_SuccessStatusCodeInResponse`.

Reply:
done